en/command_line_options.md

```diff
@@ -0,0 +1,36 @@
+# Command Line Options
+
+You can start *QGroundControl* with command line options. These are used to enable logging, run unit tests, and simulate different host environments for testing. 
+
+## Starting QGroundControl with Options
+
+You will need to open a command prompt or terminal, change directory to where **qgroundcontrol.exe** is stored, and then run it. This is shown below for each platform (using the `--logging:full` option):
+
+Windows Command Prompt:
+```bash
+cd "\Program Files (x86)\qgroundcontrol"
+qgroundcontrol --logging:full
+```
+OSX Terminal app (**Applications/Utilities**):
+```bash
+cd /Applications/qgroundcontrol.app/Contents/MacOS/
+./qgroundcontrol --logging:full
+```
+Linux Terminal:
+```bash
+./qgroundcontrol-start.sh --logging:full
+```
+
+## Options
+The options/command line arguments are listed in the table below. 
+
+Option | Description
+--- | ---
+`--clear-settings` | Clears the app settings.
```

Comment:
When might you need this? Or is it just generally useful?

Reply:
For example you may want to revert back to the application default settings after you modified them